packages/protocol/releaseData/initializationData/release4.json

```
@@ -1 +1,3 @@
-{}
\ No newline at end of file
+{
+  "GrandaMento": ["0x000000000000000000000000000000000000ce10", "10000000"]
```

If you want it to be 1%, then I think it should be like this
```suggestion
  "GrandaMento": ["0x000000000000000000000000000000000000ce10", "10000000000000000000000"]
```

a Fixed1 has 24 zeros: https://github.com/celo-org/celo-monorepo/blob/0348ffc70b99671a91314f4c345dc11eaa1e81fc/packages/sdk/utils/src/fixidity.ts#L4

anyways, I'd suggest leave TODO so we're forced to mindfully set it before the CGP, AFAIK, this is not used elsewhere

---

Oops thanks